src/status_im/contexts/wallet/events.cljs

```diff
@@ -326,55 +326,26 @@
               (assoc-in [:wallet :ui :search-address :local-suggestions] suggestion)
               (assoc-in [:wallet :ui :search-address :valid-ens-or-address?] (boolean result)))})))
 
-(rf/reg-event-fx :wallet/fetch-address-suggestions
- (fn [{:keys [db]} [_address]]
-   {:db (-> db
-            (assoc-in [:wallet :ui :search-address :local-suggestions] nil)
-            (assoc-in [:wallet :ui :search-address :valid-ens-or-address?] false))}))
-
-(rf/reg-event-fx :wallet/ens-validation-success
- (fn [{:keys [db]} [_ens]]
-   {:db (-> db
-            (assoc-in [:wallet :ui :search-address :local-suggestions] nil)
-            (assoc-in [:wallet :ui :search-address :valid-ens-or-address?] true))}))
-
 (rf/reg-event-fx :wallet/address-validation-success
- (fn [{:keys [db]} [_]]
+ (fn [{:keys [db]}]
    {:db (assoc-in db [:wallet :ui :search-address :valid-ens-or-address?] true)}))
 
+(rf/reg-event-fx :wallet/address-validation-failed
+ (fn [{:keys [db]}]
+   {:db (assoc-in db [:wallet :ui :search-address :valid-ens-or-address?] false)}))
+
 (rf/reg-event-fx :wallet/validate-address
- (fn [{:keys [db]} [address]]
```

Comment:
These events were implemented for mock purposes. Now, they're not needed anymore. So I deleted them